Ginger/GingerCoreCommon/Repository/ApplicationModelLib/AppModelParameter.cs

```diff
@@ -132,4 +132,18 @@ public string GetDefaultValue()
 
 
     }
+
+    /// <summary>
+    /// This class is used in the Export Process
+    /// </summary>
+    public class AppParameters
+    {     
+        public string ItemName { get; set; }
+
+        public ObservableList<OptionalValue> OptionalValuesList = new ObservableList<OptionalValue>();
+        
+        public string OptionalValuesString { get; set; }
+
+        public string Description { get; set; }
+    }
 }
```

Comment:
Please do not set more than 1 Class inside 1 .cs file, please take it into standalone cs file

Reply:
created separate class file for "AppParameters"